File: Array/Easy/2011. Final Value of Variable After Performing Operations.cpp
#include <iostream>
#include <vector>
using namespace std;
class Solution
{
public:
    int finalValueAfterOperations(vector<string> &operations)
    {
        int finalValue = 0;
        for (auto &it : operations)
        {
            // cout << it << ' ';
            if (it == "--X" || it == "X--")
            {
                finalValue = finalValue - 1;
            }
            else
            {
                finalValue = finalValue + 1;
            }
        }
        return finalValue;
    }
};

int main()
{
    Solution solution;
    vector<string> operations{"--X", "X++", "X++"};
    cout << solution.finalValueAfterOperations(operations) << endl; // Call the method
    return 0;
}

File: Array/Easy/2011. Final Value of Variable After Performing Operations.js
/**
 * @param {string[]} operations
 * @return {number}
 * https://leetcode.com/problems/final-value-of-variable-after-performing-operations/
 */
var finalValueAfterOperations = function (operations) {
  let finalValue = 0;
  for (let i = 0; i < operations.length; i++) {
    if (operations[i] == "--X" || operations[i] == "X--") {
      finalValue--;
    } else {
      finalValue++;
    }
  }
  return finalValue;
};
